docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,75 @@
+# Set up a database for Scalar DB/Scalar DL deployment in AKS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in AKS.
```

Comment:
Is this doc supposed to explain anything specific to AKS?
If so, we should explain. If not, why it's saying AKS?

Reply:
This document is created for database setup for Scalar DL/ Scalar DB deployment in AKS. That is why we specified AKS.

Reply:
Please answer my questions properly.
I don't get the answers from your comments.

Reply:
@feeblefakie san, 
Added answers to questions you asked. Sorry for not explaining them properly.

> Is this doc supposed to explain anything specific to AKS?

This document is created to add all supported databases for scalar DL and DB deployments in AKS.
A similar document will be added for EKS also. The [SetupDatabase.md](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/SetupDatabase.md) will become large if we add all new databases to it and it will be difficult to read. So it was decided to split the document based on AKS and EKS as we have manual deployment guides for both.

> If so, we should explain. If not, why it's saying AKS?

It is saying AKS because this document explains about fully managed databases provided by Azure ( For AKS deployments).

Reply:
As you say, yes, it talks about Azure, but does it talk anything about AKS?
The doc doesn't look, so I'm very confused.
If it doesn't say anything specific to AKS, it's weird to say AKS, isn't it?